### ISSUE TITLE
Warnings due to missing arguments

### ISSUE THREAD

**Issue by growdev** [scottopolis/edd-metrics#30]
WordPress: 4.6.1
EDD: 2.6.8
EDD Metrics: 0.4.0

Loading the metrics page by going to Downalods > Metrics causes the following warnings:

```
[02-Oct-2016 17:38:09 UTC] PHP Warning:  Missing argument 2 for EDD_Metrics_Functions::metrics_batch_1() in /Users/daniel/code/growdev/edd-metrics/includes/class-edd-metrics-functions.php on line 84
[02-Oct-2016 17:38:09 UTC] PHP Stack trace:
[02-Oct-2016 17:38:09 UTC] PHP   1. {main}() /Users/daniel/sites/shopplugins/wp-admin/admin-ajax.php:0
[02-Oct-2016 17:38:09 UTC] PHP   2. do_action() /Users/daniel/sites/shopplugins/wp-admin/admin-ajax.php:91
[02-Oct-2016 17:38:09 UTC] PHP   3. call_user_func_array:{/Users/daniel/sites/shopplugins/wp-includes/plugin.php:524}() /Users/daniel/sites/shopplugins/wp-includes/plugin.php:524
[02-Oct-2016 17:38:09 UTC] PHP   4. EDD_Metrics_Functions->metrics_batch_1() /Users/daniel/sites/shopplugins/wp-includes/plugin.php:524
[02-Oct-2016 17:38:11 UTC] PHP Warning:  Missing argument 2 for EDD_Metrics_Functions::metrics_batch_2() in /Users/daniel/code/growdev/edd-metrics/includes/class-edd-metrics-functions.php on line 125
[02-Oct-2016 17:38:11 UTC] PHP Stack trace:
[02-Oct-2016 17:38:11 UTC] PHP   1. {main}() /Users/daniel/sites/shopplugins/wp-admin/admin-ajax.php:0
[02-Oct-2016 17:38:11 UTC] PHP   2. do_action() /Users/daniel/sites/shopplugins/wp-admin/admin-ajax.php:91
[02-Oct-2016 17:38:11 UTC] PHP   3. call_user_func_array:{/Users/daniel/sites/shopplugins/wp-includes/plugin.php:524}() /Users/daniel/sites/shopplugins/wp-includes/plugin.php:524
[02-Oct-2016 17:38:11 UTC] PHP   4. EDD_Metrics_Functions->metrics_batch_2() /Users/daniel/sites/shopplugins/wp-includes/plugin.php:524
```

**Comment by scottopolis** [comment]
Looking into this

**Comment by growdev** [comment]
I've got a fix in a PR

**Comment by scottopolis** [comment]
Fixed in 526f30250c8a362c7dbb879d80cfd07568c312f6
